etc/tutorial/cylc-forecasting-suite/bin/get-observations

```diff
@@ -72,10 +83,22 @@ def get_archived_data(site_id, time):
 
 def extract_observations(data):
     """Extract the relevant observations from the weather data."""
+    try:
+        direction = ORDINATES[data['Period']['Rep']['D']]
+    except KeyError:
+        direction = '0'
+        print 'No wind direction in file so using 0'
+
+    try:
+        speed = data['Period']['Rep']['S']
+    except KeyError:
+        speed = '0'
+        print 'No wind speed in file so using 0'
+
```

Comment:
I'm not so sure about this solution as it will cause the resultant wind field to be zero which will make it look like the forecast task is not doing anything.

Then again this isn't the expected case and it's better to have no wind than a broken workflow.